packages/medusa/src/api-v2/admin/products/[id]/variants/route.ts

```diff
@@ -56,5 +58,5 @@ export const POST = async (
     throw errors[0].error
   }
 
-  res.status(200).json({ product_variant: result[0] })
+  res.status(200).json({ variant: result[0] })
```

Comment:
question: As per our recent discussion, should we return product here?

Reply:
@olivermrbl yes there is a test for it that is skipped, once enabled I'll update this